x-pack/heartbeat/monitors/browser/synthexec/enrich_test.go

```diff
@@ -84,13 +90,22 @@ func TestJourneyEnricher(t *testing.T) {
 	// on the nil data.
 	for idx, se := range synthEvents {
 		e := &beat.Event{}
-		t.Run(fmt.Sprintf("event %d", idx), func(t *testing.T) {
-			enrichErr := je.enrich(e, se)
+		stdFields.IsInline = false
```

Comment:
We really have two tests here, one for each value of `IsInline`, but reading the test it's not that apparent. The idiomatic way to do this in go would be to either:

1. Create a slice of test parameters at the top of this function (pseudocode: `[]testParameters{isInline, validator}`) and to check the results differently based on different parameters
2. Create two sub-test functions using `t.Test("isInline is false", func(...` 
3. Split this into three top level functions, two `Test(` functions, one for each scenario, and a third utility function for common code

Reply:
Addressed, Let me know if this fine.